frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -206,12 +210,22 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                     if (breakpointLength) {
                         await breakpoint(breakpointLength)
                     }
+
+                    const headers: Record<string, string> = {}
+                    if (props.accessToken) {
+                        headers.Authorization = `Bearer ${props.accessToken}`
+                    }
+
                     const blob_v2 = values.featureFlags[FEATURE_FLAGS.RECORDINGS_BLOBBY_V2_REPLAY]
                     const blob_v2_lts = values.featureFlags[FEATURE_FLAGS.RECORDINGS_BLOBBY_V2_LTS_REPLAY]
```

Comment:
```suggestion
                    const blob_v2 = values.featureFlags[FEATURE_FLAGS.RECORDINGS_BLOBBY_V2_REPLAY] || !!props.accessToken
                    const blob_v2_lts = values.featureFlags[FEATURE_FLAGS.RECORDINGS_BLOBBY_V2_LTS_REPLAY] || !!props.accessToken
```

to stop the v1 confusion here